src/atoms/Checkbox/Input.web.js

```diff
@@ -5,8 +5,8 @@ const Input = styled.input.attrs({
   type: 'checkbox',
 })`
   position: absolute;
-  width: 0px;
-  height: 0px;
+  width: 1px;
+  height: 1px;
```

Comment:
Why is this needed?

Reply:
> An element with no height or width, whether defined in HTML or CSS, is typically removed from the flow of the page, so most screen readers will not read it.

https://webaim.org/techniques/css/invisiblecontent/